### PR TITLE
Compile QField on Android using NDK r22

### PR DESCRIPTION
**Pull request by nirvn**
Let's give that a try

### CODE REVIEW

File: .github/workflows/android.yml [hunk 1]
```diff
@@ -32,6 +32,8 @@ jobs:
         id: vars
         shell: bash
         run: |
+          echo "ANDROID_NDK_HOME=/usr/local/lib/android/sdk/ndk/22.1.7171670" >> $GITHUB_ENV
```
Comment (nirvn) [inline .github/workflows/android.yml:35]:
```suggestion
          echo "ANDROID_NDK_HOME=/usr/local/lib/android/sdk/ndk/22.1.7171670" >> $GITHUB_ENV
          echo "yes" | sudo /usr/local/lib/android/sdk/tools/bin/sdkmanager --licenses
```

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 79381fd66a8a33a3de9a273c2624a8e060f8a162:
  - [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2677-3726f481e48b0a20e81fbdba5576ccdfa56eab38-arm64-android.apk)

Other architectures: [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2677-3726f481e48b0a20e81fbdba5576ccdfa56eab38-arm-android.apk), [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2677-3726f481e48b0a20e81fbdba5576ccdfa56eab38-x64-android.apk), [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2677-3726f481e48b0a20e81fbdba5576ccdfa56eab38-x86-android.apk)

File: platform/android/build.gradle.in [hunk 2]
```diff
@@ -1,4 +1,12 @@
 buildscript {
+    ext {
+        buildToolsVersion = "29.0.2"
```
Comment (m-kuhn) [inline platform/android/build.gradle.in:3]: Any chance to have these version vars dynamic?

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for fade532dc534b3f00c5bfdfe74585c7e7d038f1a:
  - [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2677-1946995d888f80266aed87897a7b0f1286ccff81-arm64-android.apk)

Other architectures: [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2677-1946995d888f80266aed87897a7b0f1286ccff81-arm-android.apk), [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2677-1946995d888f80266aed87897a7b0f1286ccff81-x64-android.apk), [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2677-1946995d888f80266aed87897a7b0f1286ccff81-x86-android.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for e5e8ea6767bb66c5396ee195cf03d7c15dd188d8:
  - [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2677-3aef85b4d2d28e3ffc76af74cf85fce0b0949154-arm64-android.apk)

Other architectures: [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2677-3aef85b4d2d28e3ffc76af74cf85fce0b0949154-arm-android.apk), [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2677-3aef85b4d2d28e3ffc76af74cf85fce0b0949154-x64-android.apk), [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2677-3aef85b4d2d28e3ffc76af74cf85fce0b0949154-x86-android.apk)

**Review by m-kuhn — APPROVED** [review]

**Comment by m-kuhn** [comment]
Are the vars properly initialized for the local docker build

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 3c6553feb4480086ac2db37c3d081cf2d2e49698:
  - [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2677-e5fa39ff802f221f3fcca5e20d0cdfd8443c0ece-arm64-android.apk)

Other architectures: [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2677-e5fa39ff802f221f3fcca5e20d0cdfd8443c0ece-arm-android.apk), [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2677-e5fa39ff802f221f3fcca5e20d0cdfd8443c0ece-x64-android.apk), [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2677-e5fa39ff802f221f3fcca5e20d0cdfd8443c0ece-x86-android.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 4dcf9f9bdacab026f9f6c5972e0f46134c35ea55:
  - [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2677-5dfbe8c3dff9aa5a82c42d9cd10c669bfa4988fa-arm64-android.apk)

Other architectures: [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2677-5dfbe8c3dff9aa5a82c42d9cd10c669bfa4988fa-arm-android.apk), [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2677-5dfbe8c3dff9aa5a82c42d9cd10c669bfa4988fa-x64-android.apk), [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2677-5dfbe8c3dff9aa5a82c42d9cd10c669bfa4988fa-x86-android.apk)

**Pull request merged**